youtube_dl_gui/optionsframe.py

```diff
@@ -607,6 +607,7 @@ class ConnectionTab(TabPanel):
 
     SPINCTRL_SIZE = (60, -1)
 
+    MAXSIMDLS_LABEL = _("Max. Simultaneous Downloads")
```

Comment:
You should change this to "Max Simultaneous Downloads" (Remove the dot from Max)